cmd/cost/get.go

```diff
@@ -369,7 +370,7 @@ func (o *getOptions) printCostGet(cost float64, unit string, ops *getOptions, OU
 	}
 
 	if ops.csv { //If csv option specified, print result in csv
-		fmt.Printf("\n%s,%f (%s)\n\n", *OU.Name, cost, unit)
+		fmt.Printf("\n%s,%s (%s)\n\n", *OU.Name, cost.StringFixed(2), unit)
```

Comment:
This just shows with decimal points? 
Not around a goenv to check

Reply:
it rounds to 2 decimal places, not sure if that answers your question?